kf_model_fhir/ingest_plugin/kids_first_fhir.py

```diff
@@ -17,10 +17,30 @@
 
 from ncpi_fhir_utility.client import FhirApiClient
 
-from kf_model_fhir.ingest_plugin.resources.kfdrc_patient import Patient
+from kf_model_fhir.ingest_plugin.target_api_builders.practitioner import Practitioner
+from kf_model_fhir.ingest_plugin.target_api_builders.organization import Organization
+from kf_model_fhir.ingest_plugin.target_api_builders.practitioner_role import PractitionerRole
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_patient import Patient
+from kf_model_fhir.ingest_plugin.target_api_builders.group import Group
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_research_study import ResearchStudy
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_patient_relations import PatientRelation
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_condition import Condition
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_phenotype import Phenotype
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_vital_status import VitalStatus
+from kf_model_fhir.ingest_plugin.target_api_builders.kfdrc_specimen import Specimen
 
 all_targets = [
+    Practitioner,
+    Organization,
+    PractitionerRole,
     Patient,
+    Group,
+    ResearchStudy,
+    # PatientRelation,
```

Comment:
why are PatientRelation and VitalStatus commented out?

Reply:
Per VitalStatus, @gsantia used SMONED CT for VitalStatus profile creation. Since the test server (as well as all other deployed servers) doesn’t pre-import and expand SMOMED, until we figure out how to use a remote terminology server, the integration test will always fail (I am still contacting Smile about this). So, I commented out VitalStatus for now. Should we add a doc-string explaining this?

Per PatientRelation, as you know PatientRelation is an extension of Patient, and before it’s running, all the reference patients should already be created. That is why I let the plugin have two separate runs for Patient and PatientRelation respectively. And since our resource HTTP request defaults to either POST or PUT, the PatientRelation run modifies patients to only having patient relations (dropping ethnicity, race, etc.). Of course, we can re-load all the attributes during the PatientRelation run, but that seems redundant work. Instead, we can definitely use PATCH, but to me that seems like a NCPI utility level discussion, so I wanted to ask @fiendish and @znatty22 opinions.